src/lib/y2network/autoinst/dns_reader.rb

```diff
@@ -40,9 +41,9 @@ def initialize(section)
       def config
         Y2Network::DNS.new(
           dhcp_hostname:      section.dhcp_hostname,
-          hostname:           section.hostname,
+          hostname:           section.hostname || default_hostname,
           nameservers:        valid_ips(section.nameservers),
-          resolv_conf_policy: section.resolv_conf_policy,
+          resolv_conf_policy: section.resolv_conf_policy || "auto",
```

Comment:
Ideally not a string, but a constant (and used elsewhere too)

Reply:
Yes, actually we might think about having an enum, but it is basically only used here. I will move it to a constant though.